### PR TITLE
Page 2 mock

### PR DESCRIPTION
**Pull request by evadb**
yeehaw

### CODE REVIEW

File: frontend/src/Page2.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+import { jsx, css } from '@emotion/core'
+import { TextArea } from './components/text-area'
+import { Button } from './components/button'
+import { Label } from './components/label'
+import { P, Lead } from './components/paragraph'
+
+/* eslint-disable-next-line */
+const submitAndNavigate = (client, data) => {
+  // client.writeData({ data })
+  // console.log('hit submit')
```
Comment (sastels) [inline frontend/src/Page2.js]: 🔥 (at least the console.log)

File: frontend/src/Page2.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+  navigate('/old/form3')
```
Comment (sastels) [inline frontend/src/Page2.js]: `/summary`

**Comment by evadb** [comment]
@sastels extreme yolo approve b4 tests happy friday

File: frontend/src/Page2.js (hunk 1 above)
Comment (evadb) [reply frontend/src/Page2.js]: just taking out the log because we'll want the other stuff later

**Pull request merged**